harmony/README.md

```diff
@@ -0,0 +1,29 @@
+# Chainlink External Adapter for Harmony
+This adapter is built to fulfill Chainlink oracle requests.
+## Configurations
+The adapter uses the following environment variables
+* HMY_NODE_URL: a URL to a JSON-RPC (HTTP RPC) node on Harmony, default: `https://api.s0.t.hmny.io`
+* HMY_CHAIN_ID: Harmony network chain id, default: `1 (mainnet)`
```

Comment:
```suggestion
* RPC_URL: a URL to a JSON-RPC (HTTP RPC) node on Harmony, default: `https://api.s0.t.hmny.io`
* CHAIN_ID: Harmony network chain id, default: `1 (mainnet)`
```

We should prevent prefixing the env var keys here, as then it's easier to prefix it if we're including this adapter in another package. Also replaced the node URL with a key we more commonly use for RPC URLs.